app/views/bulk_updates/show.html.erb

```diff
@@ -0,0 +1,41 @@
+<div class="row row-content">
+  <section>
+    <h2>
+      Bulk Update <%= @task_id %>
+      <span class="bulk-update-status eui-badge--sm"><%= @task.fetch('task-status') %></span>
+    </h2>
+    <p class="bulk-update-status-message"><%= @task.fetch('status-message', 'Status not retrieved') %></p>
+
+  </section>
+
+  <section>
+    <% unless @task.fetch('collection-statuses', nil).blank? %>
```

Comment:
Thanks. I had it like that before, I think I changed it b/c nil seems a more explicit check but what you're saying here makes more sense.